sources/src/OpenAI.tsx

```diff
@@ -20,3 +20,13 @@ export default createSvgIcon(
   'OpenAI'
 );
+
+export const ThinkingIcon = () => (
```

Comment:
This should be its own file. Also, ideally it should be a component like the other icon here created with `createSvgIcon`, but the way it's being used could make that challenging. But honestly, we should stick to the loaders in our design system and just use the CircularProgress component, so it might as well just go away.

Reply:
I made it the own file for now. I'm ok to change this to `CircularProgress` if we all agreed with it cc @russdanner

Reply:
Agreed. @phuongnq @rart

Reply:
Updated to use `CircularProgress`